p11-kit/virtual.c

```diff
@@ -3062,24 +4177,30 @@ static CK_FUNCTION_LIST *
 p11_virtual_wrap_fixed (p11_virtual *virt,
```

Comment:
We can change the return type of this function to `CK_FUNCTION_LIST_3_0`, given the PKCS#11 3.0 functions are always present.